unicef/templates/preliminary-interview-template.adoc

```diff
@@ -0,0 +1,60 @@
+= Preliminary interview template
+
+image::https://img.shields.io/badge/License-CC%20BY--SA%204.0-lightgrey.svg[License: CC BY-SA 4.0, link=https://creativecommons.org/licenses/by-sa/4.0/]
+
+This document is used as a reference when conducting interviews with the https://unicefinnovationfund.org/[UNICEF Innovation Fund] cohorts.
+The interviews help us collect background and context for each team's experience and familiarity with open source and participating in open source communities.
+This template provides some structure and guidance on what questions to ask and understand about each team.
+
+
+== Current situation
```

Comment:
It'd be good to get a quick 1-2 liner on what their product actually is. Will give more context to the below questions.